server/public/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
```

Comment:
same thing -delete/ replace with 404 page?
can we put in a '302' page or maybe an html meta refresh redirect for the two 'app store' pages?

Reply:
We can definitely do redirects. I need to read up on how exactly, so I'm going to leave it as a TODO in the `README.md`'s Development Plan section for now if that's OK.